packages/cxl-ui/src/components/cxl-paywall.js

```diff
@@ -0,0 +1,147 @@
+import '@vaadin/vaadin-dialog';
+import { customElement, html, LitElement, property, query } from 'lit-element';
+
+/**
+ * Do we want to let an unsubscribed user read the same article an unlimited amount of times?
+ * Do we want to remove content from the DOM?
+ */
+
+@customElement('cxl-paywall')
+export class CXLPaywallElement extends LitElement {

```

Comment:
Hmmm... I wonder if it makes more sense to integrate such directly into CXLAppLayout?

Reply:
They way it works right now, anything put inside of a `cxl-paywall` element generates the popup. It might make sense to isolate this functionality for simplicity reasons.